src/backend/InvenTree/order/templates/order/order_base.html

```diff
@@ -75,11 +75,11 @@
         {% endif %}
     </ul>
 </div>
-{% if order.is_pending %}
+{% if order.status == PurchaseOrderStatus.PENDING or order.status == PurchaseOrderStatus.ON_HOLD %}
```

Comment:
C: I think it would make sense to use a property on the order and move the business logic into python instead of this template.

Reply:
I have done this across the board. I was hoping not to touch the CUI too much, but I have folded these new features in now.